### DISCUSSION TITLE
[PortableApps.com] Run a PWA using Parameter

### DISCUSSION THREAD

**Discussion by xmha97** [filips123/PWAsForFirefox#289] [Q&A] [answered]
Hi @filips123 
I'm using the PortableApps.com version of PWAsForFirefox.
I created YouTube PWA and want to run this app using parameters.
I want to create several shortcuts with different parameters on the desktop.

For example, I want to launch the YouTube PWA on the search page about the "Frontend" topic:

Method 1 
```
SET AppPath4="..\FFPWA-01GQNVWPT3AP3AMS8PKRQJTJ0B\launch.vbs"
SET AppPrm4="https://www.youtube.com/results?search_query=Frontend"
START "" %AppPath4% %AppPrm4%
```

Method 2
```
SET AppPath4="..\PWAsForFirefoxPortable\App\PWAsForFirefox\firefoxpwa.exe"
SET AppPrm4=site launch 01GQNVWPT3AP3AMS8PKRQJTJ0B "https://www.youtube.com/results?search_query=Frontend"
START "" %AppPath4% %AppPrm4%
```

But these commands don't work, please help me.
Thanks.

**Comment by filips123** [comment, marked as answer]
To launch PWA on a custom URL, you need to provide `--url` parameter. This should work:

```bash
SET AppPath4="..\PWAsForFirefoxPortable\App\PWAsForFirefox\firefoxpwa.exe"
SET AppPrm4=site launch 01GQNVWPT3AP3AMS8PKRQJTJ0B --url "https://www.youtube.com/results?search_query=Frontend"
START "" %AppPath4% %AppPrm4%
```

**Reply by xmha97** [reply]
Thanks.
